src/olympia/lib/settings_base.py

```diff
@@ -1833,8 +1830,31 @@ def get_redis_settings(uri):
     'ORDERING_PARAM': 'sort',
 }
 
+
+def get_raven_release():
+    version_json = os.path.join(ROOT, 'version.json')
+    version = None
+
+    if os.path.exists(version_json):
+        try:
+            with open(version_json, 'rb') as fobj:
+                version = json.loads(fobj.read())['version']
+        except (IOError, KeyError):
+            version = None
+
+    if version is None or version == 'origin/master':
```

Comment:
`version` is `""` rather than `None`/`null` on dev https://addons-dev.allizom.org/__version__ fwiw.

Reply:
Fixed in 5abf6ff967